app/controllers/projects_controller.rb

```diff
@@ -21,12 +21,13 @@ def create
     if @project.save
       redirect_to projects_path
     else
-      render 'new'
+      redirect_to new_project_path
```

Comment:
should be render 'new', undo this change
TODO: discuss

Reply:
undo change, done